.github/workflows/msbuild.yml

```diff
@@ -15,12 +15,27 @@ jobs:
         ## Macos jobs tend to stall -- not including them for now
         ## Windows jobs fail on some lit tests and sometimes fail even to
         ## setup a work environment (e.g. dotnet installation)
-        os: [ubuntu-latest, ubuntu-16.04]
+        os: [ubuntu-latest]
         ##os: [ubuntu-latest, windows-latest, ubuntu-16.04]
+        ##os: [ubuntu-latest, macos-latest]
         ##os: [ubuntu-latest, macos-latest, windows-latest, ubuntu-16.04]
         ## Size of the following array must match --num-shards in lit command
         shard: [1, 2, 3, 4, 5]
         ##shard: [1, 2, 3]
+        include:
+        - os:  'ubuntu-latest'
+          osb: 'ubuntu'
+          rel: 'ubuntu-16.04'
```

Comment:
I'd prefer unabbreviated variable names, like `release`. Ideally we can get rid of this variable by using something like `{{ matrix.osb }}-*.zip` instead of `{{ matrix.rel }}.zip` when selecting which file to unzip

Reply:
Expanded the name. The * does not work because there are multiple releases for the same OS, e.g. at present ubuntu-14.04 and ubuntu-16.04